src/components/common/ExplorerButton/index.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ReactElement, ComponentType } from 'react'
+import { IconButton, SvgIcon, Tooltip } from '@mui/material'
+import LinkIcon from '@/public/images/common/link.svg'
+
+export type ExplorerButtonProps = {
+  title?: string
+  href?: string
+  className?: string
+  icon?: ComponentType
+}
+
+const ExplorerButton = ({ title = '', href = '', icon = LinkIcon, className }: ExplorerButtonProps): ReactElement => (
+  <Tooltip title={title} placement="top">
+    <IconButton
+      className={className}
+      target="_blank"
+      rel="noreferrer"
+      href={href}
+      size="small"
+      sx={{ color: 'inherit' }}
+    >
+      <SvgIcon component={icon} inheritViewBox fontSize="small" />
```

Comment:
The custom `Icon` component had some css set on the `SvgIcon`. Is it still covered after porting?

Reply:
It looks the same to me in the end result, but again, I'm not sure which original code you're referring to. In safe-react-components?

Reply:
Yes, I was comparing it to what we have in safe-react-components.

Reply:
True, it's actually necessary to prevent NFT previews from opening on the NFT page (https://github.com/safe-global/safe-react-components/pull/220).
I'll restore it in another PR.